Atomic/syscontainers.py

```diff
@@ -1354,6 +1353,8 @@ def prune_ostree_images(self):
         refs = {}
         app_refs = []
 
+        self.prune_storage(repo)
```

Comment:
I feel like pruning when getting the repo might be a surprise. I realize this is an internal callable so it may not be a big deal.